Rakefile

```diff
@@ -14,13 +14,13 @@ end
 desc 'Remove generated and irrelevant files'
 task :clean do
   rm_rf [
-    'bundle',
-    '.bundle',
-    'Gemfile.lock',
-    'proving_grounds',
-    'tags',
-    *Dir['*.gem'],
-  ]
+            'bundle',
+            '.bundle',
+            'Gemfile.lock',
+            'proving_grounds',
+            'tags',
+            *Dir['*.gem'],
+        ]
```

Comment:
That's a whoopsie on my part, I think I must have hit a "realign everything" button at some point.